backend/src/api/integration/helpers/twitterAuthenticateCallback.ts

```diff
@@ -5,10 +5,10 @@ import IntegrationService from '../../../services/integrationService'
 export default async (req, res) => {
   // Checking we have permision to edit the integration
   new PermissionChecker(req).validateHas(Permissions.values.integrationEdit)
-
   const urlSearchParams = new URLSearchParams(req.query.state)
   const redirectUrl = urlSearchParams.get('redirectUrl')
-  const hashtags = urlSearchParams.get('hashtags')
```

Comment:
Also, we can keep these because it'll be forwarded back in the state parameter, but since it's serialized and deserialized, we need to update the below part as 
```
  const integrationData = {
    profileId: req.user.twitter.profile.id,
    token: req.user.twitter.accessToken,
    refreshToken: req.user.twitter.refreshToken,
    hashtags: hashtags.split(","),
  }
```